src/webapi/document.rs

```diff
@@ -116,4 +116,17 @@ impl Document {
             js!( @(no_return) @{self}.title = @{title}; );
         }
     }
+
+    /// Returns the Element that is the root element of the document (for example, the <html>
+    /// element for HTML documents).
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/Document/documentElement)
+    // https://www.w3.org/TR/DOM-Level-2-Core/core.html#ID-87CD092
```

Comment:
Please link to this spec instead: https://dom.spec.whatwg.org/#ref-for-dom-document-documentelement

Ditto for other links.